app/src/main/res/values/static_strings.xml

```diff
@@ -60,4 +60,44 @@
     <!-- Secret debug info strings -->
     <string name="debug_info_region_home" translatable="false">Home region</string>
     <string name="debug_info_region_current" translatable="false">Current region</string>
+
+    <!-- Profiler settings -->
+    <string name="preferences_start_profiler">Start Profiler</string>
+    <string name="profiler_stop">Stop Profiler</string>
```

Comment:
It's hard to quickly intuit how these strings are used because of the lack of grouping. I think it'd help to add a newline here.

```suggestion
    <string name="profiler_stop">Stop Profiler</string>
    
```